app/renderer/js/pages/preference/shortcuts-section.ts

```diff
@@ -0,0 +1,341 @@
+'use strict';
+
+import { shell } from 'electron';
+
+import BaseSection = require('./base-section');
+
+class ShortcutsSection extends BaseSection {
+	// TODO: TypeScript - Here props should be object type
+	props: any;
+	constructor(props: any) {
+		super();
+		this.props = props;
+	}
+
+	templateMac(): string {
```

Comment:
Can you add a todo here briefly explaining that we need to de-duplicate the `templateMac()` and `templateWinLin()` here. Both of them have same template literal but and the only thing diffrent about them is the `userOSKey` which we could pass as argument.

Reply:
No, the template literal is not same. I just compared them. One is 157 lines, while other is 141. Still, added a comment.

Reply:
Hmm cool! Either way I suspect it majority of it is the same. Thanks for adding the comment.